### PR TITLE
Ensure unique pharmacy urls

### PR DESCRIPTION
**Pull request by chrysanthos**
This PR introduces a new field `slug`. This makes sure the seo url is always the same even if Laravel decides to change how it works at some point.

Additionally it solves the issue where you could basically visit `/678/farmakeia-giannakis` and `/678/its-me` and see the same thing since the page only depends on the am attribute. 

Some other minor refactors as well.

The main reason I needed the package is that there are pharmacies with the same name ending up having errors since slug should be unique. 

Ideally, I'd like to merge this asap before google picks the urls

### CODE REVIEW

File: app/Http/Controllers/PagesController.php [hunk 1]
```diff
@@ -1,33 +0,0 @@
-<?php
```
Comment (gregkos) [inline app/Http/Controllers/PagesController.php:1]: If we are deleting this, restructure all the views as well...

File: app/Models/Pharmacy.php [hunk 2]
```diff
@@ -82,6 +85,13 @@ public function getIsAvailableAttribute()
 
     public function getSeoUrlAttribute()
     {
-        return route('farmakeio', ['am' => $this->am, 'name' => \Str::slug($this->name)]);
+        return route('farmakeio', ['am' => $this->am, 'name' => $this->slug]);
+    }
+
+    public function getSlugOptions() : SlugOptions
```
Comment (gregkos) [inline app/Models/Pharmacy.php:91]: We need to update the coordinate filling and the single pharmacy map to use this then.

File: composer.json [hunk 3]
```diff
@@ -24,6 +25,7 @@
         "maatwebsite/excel": "^3.1",
         "spatie/browsershot": "^3.40",
         "spatie/laravel-sitemap": "^5.9",
+        "spatie/laravel-sluggable": "^2.6",
```
Comment (gregkos) [inline composer.json:28]: I get setting the slug to a db value, but did we really need a whole new package for that..?

File: routes/web.php [hunk 4]
```diff
@@ -20,17 +19,18 @@
...
-Route::get('/pharmacies', [PharmacyController::class, 'index']);
-
-Route::middleware(['auth:sanctum', 'verified'])->group(function () {
-    Route::resource('pharmacies', PharmacyController::class)->except(['index']);
-});
+Route::prefix('admin')
+    ->name('admin.')
+    ->middleware(['auth:sanctum', 'verified'])
+    ->group(function () {
+        Route::resource('pharmacies', \App\Http\Controllers\Admin\PharmacyController::class);
```
Comment (gregkos) [inline routes/web.php:32]: Import and remove FQN

**Review by gregkos — CHANGES_REQUESTED** [review]
I am okay with the idea, although the execution needs more work. Can you give me an example of pharmacies that throw errors due to having the same name? The url was based on AM only to avoid such problems. 🤔

File: app/Models/Pharmacy.php (hunk 2 above)
Comment (chrysanthos) [reply app/Models/Pharmacy.php:91]: Not sure what you mean

File: composer.json (hunk 3 above)
Comment (chrysanthos) [reply composer.json:28]: We don't really, but we are fancy and this allows to reduce our code and our bugs 😸

File: app/Http/Controllers/PagesController.php (hunk 1 above)
Comment (chrysanthos) [reply app/Http/Controllers/PagesController.php:1]: In another PR

File: app/Models/Pharmacy.php (hunk 2 above)
Comment (gregkos) [reply app/Models/Pharmacy.php:91]: Look at `pharmacies:fill-coordinates`

Comment (gregkos) [reply app/Models/Pharmacy.php:91]: And also at `single.blade.php`, but I hope that one was obvious

**Review by gregkos — CHANGES_REQUESTED** [review]

**Review by gregkos — APPROVED** [review]

**Pull request merged**